.env.example

```
@@ -0,0 +1,37 @@
+# env 템플릿
+# cp .env.example .env 로 항상 복사해서 사용할 것
+# !!! 실제 값 들어간 상태로 형상관리 되지 않도록 주의할 것 !!!
+
+# Spring 프로필
+SPRING_PROFILE=dev
+SERVER_PORT=8080
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SERVER_PORT key should go before the SPRING_PROFILE key

(UnorderedKey)